dbt/models/default/schema.yml

```diff
@@ -97,17 +114,21 @@ models:
             - mail_address_state
             - mail_address_zipcode_1
             - mail_address_zipcode_2
+          config:
+            error_if: ">879261"
```

Comment:
**issue (non-blocking)**: This high error rate is a result of using `CONCAT_WS` to construct `prop_address_full` and `mail_address_full`. Any address missing a unit number or description will have trailing whitespace. See https://github.com/ccao-data/data-architecture/blob/36a42a7d304577fd77440ff1d807905d5bae1a75/aws-athena/views/default-vw_pin_address.sql#L20C12-L20C21.

@wrridgeway Can you make a new PR to master wrapping `CONCAT_WS` in `TRIM()` for these address fields.

@jeancochrane We can keep this PR moving but we'll need to remember to update this threshold once Billy's PR is merged.